src/ducks/apps/index.js

```diff
@@ -338,11 +323,17 @@ export function fetchRegistryApps (channel = DEFAULT_CHANNEL) {
         return Promise.all(
           apps.map(app => {
             return getFormattedRegistryApp(app, channel).catch(err => {
-              console.warn(
-                `Something went wrong when trying to fetch more informations about ${
-                  app.slug
-                } from the registry on ${channel} channel, so we skip it. ${err}`
-              )
+              if (err.status === 404) {
+                console.warn(
+                  `No ${channel} version found for ${app.slug} from the registry.`
+                )
+              } else {
+                console.warn(
```

Comment:
Is there a reason here to not throw the Error ? (or at least, to `console.error` it ?)

Reply:
Yes, this is not an error actually. An application can have an empty list of versions for the default channel (stable here). Just the Store don't find any versions from this channel, it won't be displayed by the store. But it could be useful to be warned about that, couldn't it?